src/nativeORTBTrackerManager.js

```diff
@@ -0,0 +1,39 @@
+import { loadScript, triggerPixel } from "./utils";
+
+const AD_ANCHOR_CLASS_NAME = 'pb-click';
+const ASSET_ID_ELEMENT_ATTRIBUTE = 'hb_native_asset_id';
+
+export function fireNativeImpressionTrackers(ortb) {
+    const eventTrackers = ortb.eventtrackers || [];
```

Comment:
with https://github.com/prebid/Prebid.js/pull/8748, this should no longer be done here. I think that might be  better in general since it doesn't force you to use the PUC; if we still want to do that here, the logic needs to be adjusted (see that PR) _and_ we need to figure out a migration strategy so that trackers are not fired twice.

Reply:
Tracker logic was moved to PrebidJS.